.github/workflows/publish_charm.yaml

```diff
@@ -231,7 +250,14 @@ jobs:
           done
           
           for image in "${images[@]}"; do
-            resource=$(echo $image | awk -F '/' '{print $NF}' | cut -d ':' -f 1)-image
+            resource_name=$(echo $image | awk -F '/' '{print $NF}' | cut -d ':' -f 1)
+            # This is for paas-app-charmer for Django, as the main image name is named "django-app-image" and
+            # should match the main rockcraft.yaml.
+            if [[ ${resource_name} == ${CHARM_NAME} ]] && [[ " ${resources[@]} " =~ " django-app-image " ]]; then
+                resource_name=django-app
+            fi
+            resource=${resource_name}-image
```

Comment:
Could we then also add the mechanism for flask? I guess it would be `flask-app-image` and `flask-app`. Is this correct @weiiwang01?

Reply:
Yes, I think it would be good to do the same for Flask, as it has the same issue where the `name` in `rockcraft.yaml` doesn't match `resources` in `charmcraft.yaml` or `metadata.yaml`. Thanks!